server/text_generation_server/models/vlm_causal_lm.py

```diff
@@ -422,7 +430,10 @@ def forward(
             max_s = batch.max_current_length
             lm_head_indices = batch.prefill_head_indices
 
-        if self.model.config.model_type == "qwen2_vl":
+        if (
+            self.model.config.model_type == "qwen2_vl"
+            or self.model.config.model_type == "qwen2_5_vl"
+        ):
```

Comment:
```suggestion
        if self.model.config.model_type in {"qwen2_vl", "qwen2_5_vl"}:
```
Fixing the chatbot for you :)